components/folder/edits/FolderShare.tsx

```diff
@@ -0,0 +1,33 @@
+'use client'
+
+import Image from 'next/image';
+
+import { useOpenModal } from '@/hooks/useOpenModal';
+import shareImg from '@/public/folder/share.svg';
+import ModalFolderShare from '@/components/modals/ModalFolderShare';
+
+const FOLDER_SHARE_ICON = Object.freeze({
+  src: shareImg, name: '공유'
+});
```

Comment:
```suggestion
const FOLDER_SHARE_ICON = {
  src: shareImg, name: '공유'
} as const;
```

ts에서는 as const를 활용하면 읽기 전용 타입으로 선언되므로 더 제어하기 쉽습니다.(타입 좁히기도 같이 겸해줌)